inc/dbmysqliterator.class.php

```diff
@@ -618,13 +656,13 @@ private function analyzeCriterionValue($value) {
       $crit_value = null;
       if (is_array($value)) {
          foreach ($value as $k => $v) {
-            $value[$k] = DBmysql::quoteValue($v);
+            $value[$k] = $this->quoteValue($v);
```

Comment:
This cannot work. `$this` is not an instance of `DBmysql`.

Reply:
Fixed :)